onnx/defs/reduction/defs.cc

```diff
@@ -33,7 +34,10 @@ tensor has the same rank as the input if keepdims equal 1. If keepdims equal 0,
 the resulted tensor have the reduced dimension pruned.
 
 The above behavior is similar to numpy, with the exception that numpy default keepdims to
-False instead of True.)DOC";
+False instead of True.
+
+The attribute complement_axis is used in conjunction with the axis attribute, and has a default value of False.
+If complement_axis is True, all axes except the specified axis will be reduced.)DOC";
```

Comment:
Doc for all reduction ops will be updated. complement_axes related documentaiton should only be added for ReduceMax and ReduceMin